starfinder/db/models.py

```diff
@@ -164,44 +193,119 @@ class Equipment(db_engine.Model, ModelBase, HasId):
 #                                      nullable=False)
 
 
-# class Fusion(db_engine.Model, ModelBase, HasId):
-#     attributes = db_engine.Column(db_engine.JSON("fusions.attributes"),
-#                                      nullable=False)
+class Fusion(db_engine.Model, ModelBase, HasId):
+    id = db_engine.Column(db_engine.Integer(), primary_key=True)
+    name = db_engine.Column(db_engine.String(64), nullable=False)
+    level = db_engine.Column(db_engine.Integer(), nullable=False)
+    description = db_engine.Column(db_engine.String(64), nullable=False)
 
+class TechItem(db_engine.Model, ModelBase, HasId):
+    type_tech_item = db_engine.Column(db_engine.String(64), nullable=False)
```

Comment:
this is a weird name... I don't remember what this is supposed to be referencing. would `item_type` be more accurate?